sdk/search/azure-search-documents/azure/search/documents/_generated/models/_models_py3.py

```diff
@@ -1865,7 +1865,7 @@ def __init__(self, **kwargs: Any) -> None:
 
 
 class SearchScoreThreshold(VectorThreshold):
-    """The results of the vector query will filter based on the '@search.score' value. Note this is
+    """The results of the vector query will filter based on the '\@search.score' value. Note this is
```

Comment:
You should double backslash these to avoid `SyntaxWarnings` in Python versions 3.12+
```suggestion
    """The results of the vector query will filter based on the '\\@search.score' value. Note this is
```

Otherwise, users will see `SyntaxWarning: invalid escape sequence '\@'`. I had a similar [issue ](https://github.com/Azure/azure-sdk-for-python/issues/39033) in azure-monitor-query where these warnings would be produced due to improper escaping.